upgrading/automated_upgrades.adoc

```diff
@@ -221,7 +221,7 @@ If you have an older format installation configuration file in
 cluster version, the quick installer will attempt to upgrade the file to the new supported
 format. If you do not have an installation configuration file of any format, you
 can
-xref:../../install_config/install/quick_install.adoc#defining-an-installation-configuration-file[create one manually].
+xref:../install_config/install/quick_install.adoc#defining-an-installation-configuration-file[create one manually].
 
 To start an upgrade with the quick installer:
 
```

Comment:
include::install_config/upgrading/automated_upgrades.adoc[tag=automated_upgrade_after_reboot]

renders as:

Unresolved directive in <stdin> - include::install_config/upgrading/automated_upgrades.adoc[tag=automated_upgrade_after_reboot]

Reply:
Fixt